examples/acceleration.py

```diff
@@ -0,0 +1,5 @@
+from adafruit_circuitplayground.express import cpx
```

Comment:
I would add an `import time` and `time.sleep(0.1)` at least into this one so it is more reasonable to read.

Reply:
ok

Reply:
ok